util/math/math.go

```diff
@@ -54,3 +56,48 @@ func StrLenOfInt64Fast(x int64) int {
 func IsFinite(f float64) bool {
 	return !math.IsNaN(f - f)
 }
+
+type ordered interface {
+	~int | ~int8 | ~int16 | ~int32 | ~int64 | // sign
+		~uint | ~uint8 | ~uint16 | ~uint32 | ~uint64 | ~uintptr | // unsign
+		~float32 | ~float64 | // float
+		~string
+}
+
+// Max is to return the largest integer value.
+func Max[v ordered](a, b v) v {
+	if a > b {
+		return a
+	}
+	return b
+}
+
+// MaxMulti choice biggest integer from its arguments.
+func MaxMulti[v ordered](x v, xs ...v) v {
+	max := x
+	for _, n := range xs {
+		if n > max {
+			max = n
+		}
+	}
+	return max
+}
```

Comment:
why not merge the 2 methods? I think the multi version can accept 2 arguments too.

Reply:
Done